pkg/asset/manifests/openshift.go

```diff
@@ -157,6 +157,7 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		cloudCreds = cloudCredsSecretData{
 			VSphere: &VSphereCredsSecretData{
 				VCenter:              installConfig.Config.VSphere.VCenter,
+				Port:                 installConfig.Config.VSphere.Port,
```

Comment:
AFAIK we do not need to add port to the credentials. We need to add port to the `global` and `workspace` sections of the cloud provider config: https://github.com/openshift/installer/blob/master/pkg/asset/manifests/vsphere/cloudproviderconfig.go